### PR TITLE
docs: Introduce guide webrtc server and client 

### PR DESCRIPTION
**Pull request by gabrielmatau79**
Resolve issue #14 

Introduce concepts and documentation about the protocol handled by the Mediasoup WebRTC server for its clients. Additionally, a guide is provided on how to build a Mediasoup client using the pymediasoup and mediasoup-client libraries, and not give up in the process.

### CODE REVIEW

File: docs/mediasoup-server-protocol.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+## Request Methods allows Overview
+
+### 1. **getRouterRtpCapabilities**
+   - **Purpose**: Retrieves the RTP capabilities of the Mediasoup router.
+   - **Usage**: Helps verify codec compatibility between client and server.
+   
+   **Example**:
+   ```javascript
+   const capabilities = await socket.request('getRouterRtpCapabilities');
+   console.log(capabilities);
```
Comment (lotharking) [inline docs/mediasoup-server-protocol.md:83]: I think console log is not necessary

File: docs/mediasoup-server-protocol.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+   await socket.request('join', {
+       displayName: 'User',
+       device: { name: 'Browser', version: 'v1' },
+       rtpCapabilities: device.rtpCapabilities
+   });
+   ```
+
+### 3. **createWebRtcTransport**
+   - **Purpose**: Creates a WebRTC transport for sending or receiving media.
+   - **Usage****: Establishes transport necessary for WebRTC media streams.
```
Comment (lotharking) [inline docs/mediasoup-server-protocol.md:101]:
```suggestion
   - **Usage**: Establishes transport necessary for WebRTC media streams.
```

File: docs/webrtc-client-setup-guide.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+- npm/yarn (for dependency management)
+- Web browser with WebRTC support (e.g., Chrome, Firefox)
+
+### Python Requirements
+- Python (v3.10 or later)
+- pip for installing dependencies
+- WebSocket server for signaling
+
+### Server Requirements
+- Mediasoup Server
```
Comment (lotharking) [inline docs/webrtc-client-setup-guide.md:20]: is there any version?

Comment (gabrielmatau79) [reply docs/webrtc-client-setup-guide.md:20]: Yes! Webrtc-server Version 1.0.0

**Pull request merged**